npm/metrics/ipset-inventory.go

```diff
@@ -16,16 +16,14 @@ func GetIPSetInventory(setName string) int {
 	return 0
 }
 
-// SetIPSetInventory sets the number of entries in an IPSet and updates a Prometheus metric.
-func SetIPSetInventory(setName string, val int) {
-	_, exists := ipsetInventoryMap[setName]
-	if exists || val != 0 {
-		ipsetInventoryMap[setName] = val
-		updateIPSetInventory(setName)
-	}
+// RemoveFromIPSetInventory deletes the setName from the IPSetInventory metric.
+// When val is 0, the set will be deleted from the IPSetInventory metric.
+func RemoveFromIPSetInventory(setName string) {
```

Comment:
I see you want to reuse `updateIPSetInventory` function.
But, it seems below code may be clear to me since it is obvious code path?
```go
labels := GetIPSetInventoryLabels(setName)
IPSetInventory.Delete(labels)
delete(ipsetInventoryMap, setName)
```
What do you think?

Reply:
I think I'll rework this code when I combine NumIPSetEntries and IPSetInventory